karate-core/src/main/java/com/intuit/karate/UICookieUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.intuit.karate;
+
+import com.intuit.karate.http.Cookie;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Cookie Utility method for UI calls.
+ */
+public class UICookieUtils {
+
+    /**
+     * creates a map corresponding to cookie. This is used by appropriate drivers.
+     * Karate Cookie Map is of type <String, String> vs the internal framwork expects appropriate types -
+     * for ex boolean for secure and persistent keys.
```

Comment:
yes this makes sense. can you just apply IDE java formatting to add white-space (nit pick)

Reply:
done!